website/src/components/table/table.tsx

```diff
@@ -80,6 +82,16 @@ const CSTable: React.FunctionComponent<{
     });
   };
 
+  if (!userData.firstName) {
```

Comment:
I can't remember how we implemented this but is it possible for the first name to be set but the graduation year to not be set?

Reply:
yes I have tested that and that is the case. you don't need a grad year

Reply:
Should I have it check for name and grad year? Or is just checking for name enough?

Reply:
grad year would be great